src/WorkplaceChannel.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace NotificationChannels\Workplace;
+
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\Exception\ClientException;
+use Illuminate\Notifications\Notification;
+use NotificationChannels\Workplace\Exceptions\CouldNotSendNotification;
+
+class WorkplaceChannel
+{
+    /** @var ClientInterface Http client. */
+    protected $httpClient;
+
+    public function __construct(ClientInterface $httpClient)
+    {
+        $this->httpClient = $httpClient;
+    }
+
+    /**
+     * Send the given notification.
+     *
+     * @param mixed $notifiable
+     * @param \Illuminate\Notifications\Notification $notification
+     *
+     * @throws \NotificationChannels\Workplace\Exceptions\CouldNotSendNotification
+     */
+    public function send($notifiable, Notification $notification)
+    {
+        if (! $to = $notifiable->routeNotificationFor('workplace', $notification)) {
+            throw CouldNotSendNotification::endpointNotProvided();
+        }
+
+        $message = $notification->toWorkplace($notifiable);
+
+        $body = [
+            'message' => $message->getContent(),
+            'formatting' => ($message->isMarkdown() ? 'MARKDOWN' : null),
```

Comment:
Minor but i'd chuck `MARKDOWN` in a const

Reply:
Done